Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
As comparison:
Old Stats looks like this:
![a](http://i.imgur.com/rgslZxl.png)
Old Sidebar looks like this:
![a](http://i.imgur.com/j9t0KwC.png)
If you really think a notice is necessary for this, comment here and I'll gladly do it.